gtdynamics/utils/WalkCycle.cpp

```diff
@@ -44,8 +45,7 @@ std::map<string, Point3> WalkCycle::initContactPointGoal(
       // If no goal set yet, add it here
       if (cp_goals.count(link_name) == 0) {
         LinkSharedPtr link = robot.link(link_name);
-        auto foot_w = link->bMcom() * kv.second.point +
-                      Point3(0, 0, -ground_height);
+        const Point3 foot_w = link->bMcom() * kv.second.point + adjust;
```

Comment:
auto probably created an Eigen Expression that did the wrong thing in combination with emplace below. Very evil!